.gitignore

```diff
@@ -322,6 +322,8 @@ src/react-apps/applications/runtime/dist
 ######### Built react apps ###########
 src/AltinnCore/Designer/wwwroot/designer/css/react
 src/AltinnCore/Designer/wwwroot/designer/js/react
+src/AltinnCore/Runtime/wwwroot/runtime/css/react
+src/AltinnCore/Runtime/wwwroot/runtime/js/react
```

Comment:
Should group Designer-stuff together, and then Runtime-stuff together, instead of having Runtime files in the middle of Designer files.